policyengine_us/variables/gov/states/ri/tax/income/ri_property_tax_household_income.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class ri_property_tax_household_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = (
+        "Rhode Island total household income for the property tax computation"
+    )
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20RI-1040H_v2_w.pdf#page=2"
+    defined_for = StateCode.RI
+    unit = USD
+    definition_period = YEAR
+    adds = ["ri_household_income"]
+    subtracts = ["above_the_line_deductions"]
```

Comment:
Where is the reference for this? Add a line number, and also add a legal code reference if available.

Reply:
Hi Max, line number is added and fail to find directly related legal code. Plz let me know if anything else needs to be added!